libs/datasets/sources/cdc_testing_dataset.py

```diff
@@ -87,6 +140,4 @@ def modify_dataset(ds: MultiRegionDataset) -> MultiRegionDataset:
 
     ts_copy = ts_copy.append(dc_results, verify_integrity=True).sort_index()
 
-    return dataclasses.replace(
-        ds, timeseries=remove_trailing_zeros(ts_copy), timeseries_bucketed=None
-    )
+    return dataclasses.replace(ds, timeseries=ts_copy, timeseries_bucketed=None)
```

Comment:
Hrm...  I guess you moved the remove_trailing_zeros() call to each of the individual scraper datasources.  I guess I can theoretically see that being useful (if the historical dataset had trailing zeros that we wanted to replace with data from the as-original one).  But I'm curious if you actually ran into a problem in practice?

Per my other comment, if we call modify_dataset() in each individual source, maybe we can undo this change?

Reply:
yep, good call! That was my thought process (removing any trailing zeros so that they would be replaced by newer data) but I didn't notice that actually happening. With the above changes this was reverted